docs/resources/password.md

```diff
@@ -63,6 +63,8 @@ resource "aws_db_instance" "example" {
 Import is supported using the following syntax:
 
 ```shell
-# Random Password can be imported by specifying the value of the string:
+# Random Password can be imported by specifying the value of the string.
+# `lower`, `upper`, `special`, `number` and `numeric` will be set to `true`.
+# `min_lower`, `min_upper`, `min_special` and `min_numeric` will be set to `0`.
```

Comment:
Suggestion: wouldn't it make more sense to say `lower`, `upper`, ... will be set to their respective default values?

Even better, could just say "every other attribute would be set to their default value, as they have one defined in the schema above".